frontend/public/components/nav/perspective-nav.tsx

```diff
@@ -56,6 +53,17 @@ const PerspectiveNav: React.FC<{}> = () => {
     return <AdminNav />;
   }
 
+  const getLabelForResource = (resource: string): string => {
+    const model: K8sKind | undefined = modelFor(resource);
+    if (model) {
+      if (model.labelPluralKey) {
+        return t(model.labelPluralKey);
+      }
+      return model.labelPlural || model.plural;
```

Comment:
Just a query, maybe in a later Bug/Feature should we not update the `labelPlural` and `plural` and add corresponding 
`// t(label)` entries in our model files just like we do for `labelKey` and `label`